Mathlib/Util/PPOptions.lean

```diff
@@ -29,3 +29,17 @@ def getPPBinderPredicates (o : Options) : Bool :=
   o.get pp.mathlib.binderPredicates.name (!getPPAll o)
 
 end Mathlib
+
+
+/--
+The `pp.mdata` option is used to control whether `Expr.mdata` nodes are pretty-printed.`
+-/
+register_option pp.mdata : Bool := {
```

Comment:
Let's move this definition into the `Mathlib` namespace. This defines a global `pp.mdata`.

I also highly recommend that this be called `pp.mathlib.mdata` like with `pp.mathlib.binderPredicates`. There's no guarantee that core won't have `pp.mdata` in the future, but there's a guarantee it won't have any options with "mathlib" in them.